cypress/integration/jsTaskParseListDOM.spec.js

```diff
@@ -14,15 +14,14 @@ describe('Parse list app ', () => {
   });
 
   it('should be sorted by salary DSC', () => {
-
     cy.get('ul > li').then(($ul) => {
-      const list = [...$ul].map(
-          (row) => row.dataset.salary.replace('$', '').replace(',', ''));
+      const list = [...$ul].map((row) =>
+        row.dataset.salary.replace('$', '').replace(',', ''),
+      );
       let counter = 0;
 
       for (let i = 0; i < list.length; i++) {
```

Comment:
Instead of mapping through the DOM elements and then iterating over them, you could directly reduce them to a single boolean value that indicates if the list is sorted correctly.